--- node0.config.toml	2025-09-13 10:16:52
+++ normal_config.toml	2025-09-13 12:54:42
@@ -19,7 +19,7 @@
 proxy_app = "tcp://127.0.0.1:26658"
 
 # A custom human readable name for this node
-moniker = "node0"
+moniker = "node1"
 
 # Database backend: badgerdb | goleveldb | pebbledb | rocksdb | cleveldb | boltdb
 # * badgerdb (uses github.com/dgraph-io/badger)
@@ -264,7 +264,7 @@
 seeds = ""
 
 # Comma separated list of nodes to keep persistent connections to
-persistent_peers = "a37be2498e5b7de9f473637c50ff1e9d9d4239ab@node0:26656,10ba995764b87431c55505be1c71c81735cb9142@node1:26656,ed4132217f0fa3f0af20c05376f42e83f04a2442@node2:26656,45613cc54e2e21c6bcc32cf29c756fcc7b5eaca2@node3:26656"
+persistent_peers = "803a0b1ff20f1fbc20670614527078877a3110ee@node0:26656,6c48bc38cee5dc937dfda16147e8f0cb3e4000f4@node1:26656,e9bcbc96e78b2a61a980a627e643a4dcac364eb2@node2:26656,b06b0de6942483daab51a2816a84afb4139e1853@node3:26656"
 
 # Path to address book
 addr_book_file = "config/addrbook.json"
@@ -286,7 +286,7 @@
 persistent_peers_max_dial_period = "0s"
 
 # Time to wait before flushing messages out on the connection
-flush_throttle_timeout = "200ms"
+flush_throttle_timeout = "10ms"
 
 # Maximum size of a message packet payload, in bytes
 max_packet_msg_payload_size = 1024
@@ -464,7 +464,7 @@
 # height (this gives us a chance to receive some more precommits, even
 # though we already have +2/3).
 # Set to 0 if you want to make progress as soon as the node has all the precommits.
-timeout_commit = "2s"
+timeout_commit = "1s"
 
 # Deprecated: set `timeout_commit` to 0 instead.
 skip_timeout_commit = false
@@ -480,7 +480,7 @@
 create_empty_blocks_interval = "0s"
 
 # Reactor sleep duration parameters
-peer_gossip_sleep_duration = "400ms"
+peer_gossip_sleep_duration = "100ms"
 peer_gossip_intraloop_sleep_duration = "0s"
 peer_query_maj23_sleep_duration = "2s"
 
